README.md

```diff
@@ -36,72 +36,87 @@
 
 This is an experimental web client for Jellyfin based on Vue.js. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start.
 
-## Requirements
+# Build Process
 
-This project requires [Node 16](https://nodejs.org/en/) and uses **npm** as a package manager.
+## Dependencies
 
-You also need the following Jellyfin server version:
+- [Node.js LTS](https://nodejs.org/en/download) `>=16.13.0 <17.0.0`
+- npm `>=8.1.0` (included in Node.js)
+- Jellyfin Server `>=10.7.0`
 
-```
-Jellyfin >=10.7.0
-```
+## Getting Started
 
-## Contributing
+1. Clone or download this repository.
 
-We provide a [devcontainer](https://code.visualstudio.com/docs/remote/containers) to help you setup your environment.
+   ```bash
+   git clone https://github.com/jellyfin/jellyfin-vue.git
+   cd jellyfin-vue
+   ```
 
-The project also contains recommended extensions for [Visual Studio Code](https://code.visualstudio.com/), which will help you with syntax style and development.
+2. Install the build dependencies in the project directory.
 
-Finally, we provide useful pre-commit hooks via [Husky](https://typicode.github.io/husky/#/), as well as [Comitizen](https://github.com/commitizen/cz-cli) integration, in order to help you respect the style and naming conventions used throughout this project.
+   ```bash
+   npm install
+   ```
 
-For more information about how to contribute to this project, see [CONTRIBUTING.md](https://github.com/jellyfin/jellyfin-vue/blob/master/CONTRIBUTING.md)
+3. Run the web client with Nuxt as a server for local development.
 
-## Build Process
+   ```bash
+   npm start
+   ```
 
-```bash
-# install dependencies
-$ npm install
+   The client will be served at http://127.0.0.1:3000 by default.
```

Comment:
```suggestion
   The client will be available at http://127.0.0.1:3000 by default.
```

I'd use this instead of served, cause really it's served on 0.0.0.0 ^^